Neos.Kickstarter/composer.json

```diff
@@ -7,6 +7,8 @@
         "php": "^7.3 || ^8.0",
         "neos/flow": "*",
         "neos/fluid-adaptor": "*",
+        "neos/fusion-afx": "*",
```

Comment:
Maybe (in another PR) we should make fluid-adaptor and fusion-afx/form optional dependencies that need to be installed manually when running the command. (see also #2152)

Reply:
To be fair, could the requirement of fluid and fusion be removed from the kickstarters composer.json and only added to the generated package composer.json?

Reply:
Not really, but even if - what would be the benefit for the end-user? It would only install in the dev-collection then. My optimal UX would be this:
- don't install any of the two above by default
- suggest to install either to generate views/templates
- if generating templates and the matching package isn't installed, install it automatically (or at least give a message that states exactly what to do)

Reply:
Kickstart can be installed without the dev-collection? and we removed Fluid as a hard dependency from that collection to.

 * The Fluid template could be build as a DomDocument with native PHP?
 *  The Fusion could be build from a template with the SimpleTemplateView class?

And when content for the kickstarted package is created, we set the requirements in the composer.json accordingly to what was chosen for generating templates (fusion or fluid)?

Reply:
@albe Do not let me comment block the merging of this :) Martin added a positive review - UX of kickstarter can be a separate topic other day 👍

Reply:
Got it, you were talking about the dependency for the template creating template language - which is Fluid. Yeah, let's check that in a separate PR :)